azurerm/helpers/azure/api_management.go

```diff
@@ -22,19 +22,19 @@ func SchemaApiManagementDataSourceName() *schema.Schema {
 	}
 }
 
-func SchemaApiManagementProductName() *schema.Schema {
+func SchemaApiManagementChildName() *schema.Schema {
```

Comment:
I assume by child it applies to all sub resources, a comment explaining this might be handy

Reply:
documented this